pkg/etcdcli/health.go

```diff
@@ -201,6 +204,23 @@ func HasStarted(member *etcdserverpb.Member) bool {
 	return true
 }
 
+// IsQuorumFaultTolerant checks the current etcd cluster and returns true if the cluster can tolerate the
+// loss of a single etcd member. Such loss is common during new static pod revision.
+func IsQuorumFaultTolerant(memberHealth []healthCheck) bool {
+	totalMembers := len(memberHealth)
+	quorum := totalMembers/2 + 1
+	healthyMembers := len(GetHealthyMemberNames(memberHealth))
+	switch {
+	case totalMembers-quorum < 1:
+		klog.Errorf("etcd cluster has quorum of %d which is not fault tolerant", quorum)
```

Comment:
won't printing the memberHealth help later on to analyze? Those clauses should not happen to often, right?

Reply:
I agree